.eslintrc.js

```diff
@@ -53,6 +53,8 @@ module.exports = {
     'one-var-declaration-per-line': 'off',
     'one-var': 'off',
     'no-constant-condition': 'off',
+    'max-classes-per-file': 'off',
+    'arrow-parens': [2, "as-needed", { "requireForBlockBody": true }],
```

Comment:
@koraa instead of changing the rules, it's better to adjust the code.
also, please use the new `@adobe/eslint-config-helix`